Source/Classes/_classes.json

```diff
@@ -1,5 +1,5 @@
 {
-   "outputFile": "vsteam.classes.ps1",
```

Comment:
I am not ok with these changes right now. These change the build process and I am not wanting to change the build process at this time.

Reply:
Understood. See my comment on the the other PR about why you might want to keep in mind for another time